src/plugins/vis_type_vega/public/data_model/vega_parser.ts

```diff
@@ -220,6 +235,8 @@ The URL is an identifier only. OpenSearch Dashboards and your browser will never
         contains: string;
       };
       useResize = Boolean(autosize?.type && autosize?.type !== 'none');
+    } else if (showPointInTimeEvents) {
+      autosize = undefined;
```

Comment:
Is this `else if` sufficient? It seems like if `auotsize` is defined, we'll respect it _even if_ `showPointInTimeEvents` is trrue. That works today because the spec is always autogenerated, but seems a bit brittle.

Reply:
This was added here because it is intercepting right before hitting the default case of setting `autosize` to be `defaultAutosize`. It needs to be set as undefined otherwise it breaks when compiling into a vega spec. See comment on line 201 + comment on [line 77 in vega_view](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/feature/feature-anywhere/src/plugins/vis_type_vega/public/vega_view/vega_view.js#L77)